brianlong/www.validators.app
# 186587956 Don't display avatars from avatar_url

#### What's this PR do?
- avatar_urls are not displayed as images anywhere

#### How should this be manually tested?
- check each page that should display validator avatars
- make sure that the sources of those avatars are not external (i.e. not validator_url field from validator model)
- run `rails r daemons/leader_stats_mainnet_update_daemon.rb` and confirm it's working on home page

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186587956)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
